pkgs/development/libraries/libdicom/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, uthash, meson, ninja, cmake, check, pkg-config }:
```

This is completely optional, but it often makes it a bit easier for reviewers to read the diffs if these arguments are laid out with newlines

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, uthash
, meson
, ninja
, cmake
, check
, pkg-config
}:
```

---

Did you push the change ?